### PR TITLE
Add Accordion component

### PR DESCRIPTION
**Pull request by mi-yu**
image
image

### CODE REVIEW

File: packages/client/src/components/Accordion/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ *       <AccordionRow title="First row">
+ *          Some content here
+ *       </AccordionRow>
+ *       <AccordionRow title="Second row">
+ *          more content
+ *       </AccordionRow>
+ *     </Accordion>
+ *   )
+ * }
+ */
```
Comment (oupadhyay) [inline packages/client/src/components/Accordion/index.js:24]: Very interesting idea. I guess it is already in the `App.js` but ok. :+1:

File: packages/client/src/components/Accordion/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+const AccordionRowHeader = styled.div`
+  height: auto;
+  cursor: pointer;
+  padding: 10px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
[the remark this answers is not in the document]
Comment (mi-yu) [reply packages/client/src/components/Accordion/index.js:32]: ah this is for vertically aligning the elements in the row headers, not the content

[the remark this answers is not in the document]
Comment (oupadhyay) [reply packages/client/src/components/Accordion/index.js:32]: So the titles should be centered? As shown in the images above, I see that the text inside is centered and the header is left-aligned. Is that what you want?

[the remark this answers is not in the document]
Comment (mi-yu) [reply packages/client/src/components/Accordion/index.js:32]: yeah the content inside is arbitrary and can be styled arbitrarily

**Comment by mi-yu** [comment]
woops missed this, yeah this will stretch to fill the width of the container, not too sure exactly how it will be used so i'll merge this for now and we can tweak the details once it gets used

**Pull request merged**